src/clj/rems/api/schema.clj

```diff
@@ -101,10 +101,7 @@
    (s/optional-key :workflow) s/Any
    (s/optional-key :possible-commands) #{s/Keyword}
    (s/optional-key :deciders) #{s/Str}
-   (s/optional-key :commenters) #{s/Str}
-   (s/optional-key :form-contents) s/Any
-   (s/optional-key :submitted-form-contents) s/Any
-   (s/optional-key :previous-submitted-form-contents) s/Any})
+   (s/optional-key :commenters) #{s/Str}})
```

Comment:
so where will the form contents come from? is this refactor here to make room for the new `:answers` api?

Reply:
They come from here: https://github.com/CSCfi/rems/blob/4d46dc7de71651b120da4a3de31ca42d428cd68b/src/clj/rems/api/schema.clj#L56-L57

The `:form-contents` et al. are never used in the UI. They are only used to calculate the field's `:value` and `:previous-value`.

Reply:
oh right, they're one level up on the application api